### DISCUSSION TITLE
Candidate list font can't display kanji/kana

### DISCUSSION THREAD

**Discussion by poettig** [google/mozc#566] [Q&A] [answered]
Hi,

I have a problem with the candidate list of mozc when using it with ibus.
I use it with mozc-ut and have noto-cjk-fonts installed. As you can see, the preview of ibus can display the characters correctly.
Using mozc via fcitx5 showed the candidate list correctly. Unfortunately, I have no idea where to look further for debugging. If someone has a tip for me, I'd appreciate it :)

![image](https://user-images.githubusercontent.com/19955832/161612985-1dd19690-0ea6-4b9c-a7a8-f05540578cf3.png)

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Hi,

Thank you for the screenshot. It's very helpful.
From the screenshot, I guess you use Qt based candidate window built with Bazel.

I guess it is a Qt font configuration issue on your environment.

Here are some information for debugging.
* Code location: https://github.com/google/mozc/tree/master/src/renderer/qt
* Entry point (main function): https://github.com/google/mozc/blob/master/src/renderer/mozc_renderer_main.cc
* Qt initialization: https://github.com/google/mozc/blob/master/src/renderer/qt/qt_server.cc
* Main logic: https://github.com/google/mozc/blob/master/src/renderer/qt/qt_window_manager.cc

`//renderer:mozc_renderer` is the build target name.
Details are described in https://github.com/google/mozc/blob/master/docs/build_mozc_in_docker.md

Thanks,

**Comment by poettig** [comment]
Thank you for all this information! With the hint that this is the Qt candidate window I was able to find the correct configuration tool  (Qt settings) and could change the font to one that works correctly :)
